src/test/groovy/com/stormpath/tck/errors/ErrorsIT.groovy

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stormpath.tck.errors;
+
+import com.jayway.restassured.http.ContentType;
+import com.stormpath.tck.AbstractIT;
+import org.testng.annotations.Test;
+
+import static com.jayway.restassured.RestAssured.given
+import static com.stormpath.tck.util.FrameworkConstants.MeRoute;
+import static com.stormpath.tck.util.FrameworkConstants.MissingRoute
+import static org.hamcrest.CoreMatchers.containsString;
+
+@Test
+class ErrorsIT extends AbstractIT {
+
+    /**
+     * A missing endpoint should return HTML when Accept header is text/html
+     * @see <a href="https://github.com/stormpath/stormpath-sdk-java/issues/706">#706</a>
+     */
+    @Test(groups=["v100", "html"])
+    public void missingEndpointShouldReturn404AsHTML() {
```

Comment:
The first two tests should not be in the framework TCK. They should fall through to the underlying framework, which behavior is undefined in the TCK's terms.

https://github.com/stormpath/stormpath-framework-spec/blob/master/requests.md